recipes-bsp/imx-sc-firmware/imx-sc-firmware_1.2.10.1.bb

```diff
@@ -35,4 +35,4 @@ INHIBIT_PACKAGE_STRIP = "1"
 INHIBIT_PACKAGE_DEBUG_SPLIT = "1"
 PACKAGE_ARCH = "${MACHINE_ARCH}"
 
-COMPATIBLE_MACHINE = "(mx8qm|mx8qxp)"
+COMPATIBLE_MACHINE = "(mx8qm|mx8x)"
```

Comment:
Why this was changed? it seems wrong. If so, please explain on commit log.

Reply:
I don't think is wrong. I used that one internally to match another machine, not upstream yet. I'll change it back

Reply:
Not upstreamed yet is wrong. Also it dropped a valid one.

Reply:
Not upstreamed because is not ready yet for public.
I just checked 
`./conf/machine/imx8qxpmek.conf:6:MACHINEOVERRIDES =. "mx8:mx8x:mx8qxp:"`
But anyway, I'll change it back to `mx8qxp`